organizers/2024-12-17.html

```diff
@@ -0,0 +1,165 @@
+
+<section>
+    <img src="/img/django-london.svg" style="width: 40%; background: none" alt="Django London" class="plain" />
+    <h2>December 17th, 2024</h2>
+    <h2>Talks starting at 19:15!</h2>
+</section>
+
+<section>
+    <h1>👋 Welcome All!</h1>
+</section>
+
+<section>
+    <h1>Code of conduct</h1>
+    <h2>😃 Be Awesome to Each Other</h2>
+    <p>
+        The mission of The London Django Meetup Group is to provide a fun, welcoming and
+        professional environment for a diverse group of people.
+    </p>
+    <p>
+        <a href="https://www.djangolondon.com/code-of-conduct/"
+            >djangolondon.com/code-of-conduct</a
+        >
+    </p>
+</section>
+
+<section>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Kraken Tech</strong></li>
+        <li><strong>Jetbrains</strong></li>
+    </ul>
+</section>
+
+<section>
+    <img
+        data-src="/img/2022_KrakenV2_stacked_White.png"
+        style="width: 60%; border-radius: 10px"
+        alt="Kraken Tech"
+        class="plain"
+    />
+    <p>
+        <em>
+            Kraken is a revolutionary energy tech platform built with Django. Serving 25
+            million households worldwide.
+        </em>
+    </p>
+</section>
+
+<section>
+    <img
+        data-src="/img/jetbrains.svg"
+        style="width: 60%; background: none;"
+        alt="Jetbrains"
+        class="plain"
+    />
+    <p>
+        <em>
+            Pro tools created with {developers}, teams, and businesses in mind.
+        </em>
+    </p>
+    <p>(They make PyCharm.)</p>
+</section>
+
+<section>
+    <h1>📋 Today's Schedule</h1>
+    <h2>🗞️ News</h2>
+    <h2>🤔 Quiz</h2>
+    <h2>🎨 Sticker table</h2>
+    <h2>💬 Open Discussion</h2>
+</section>
+
+<section>
+    <h2>🗞️ News</h2>
+    <p>From <strong>Django News</strong>, a weekly newsletter on all things Django.</p>
+    <p><a href="https://django-news.com">django-news.com</a></p>
+</section>
+
+<section>
+    <h2>💰 Django Fundraising</h2>
+    <p>Goal: $200,000</p>
+    <p>93.8% funded</p>
+    <p><a href="https://www.djangoproject.com/fundraising">djangoproject.com/fundraising</a></p>
+</section>
+
+<section>
+    <h2><code>CompositePrimaryKey</code></h2>
+    <p>19-year-old ticket fixed, for Django 5.2</p>
+    <pre class="lang-python"><code>from django.db import models
+
+class Release(models.Model):
+    pk = models.CompositePrimaryKey("version", "name")
+    version = models.IntegerField()
+    name = models.CharField(max_length=20)</code></pre>
+    <p>Thanks to Csirmaz Bendegúz for contributing through GSoC and Lily Foote for mentoring.</p>
+</section>
+
+<section>
+    <h2></h2>
+</section>
+
+<section>
+    <h2>🇮🇪 DjangoCon Europe 2025</h2>
+    <p>April 23-27</p>
+    <p>Dublin, Ireland</p>
+    <p>Tickets on sale</p>
+    <p>Call for Proposals until 12th January</p>
+
+    <p>
+        <a href="https://2025.djangocon.eu">2025.djangocon.eu</a>
+    </p>
+</section>
+
```

Comment:
<section>
    <h2>🇮🇹 PyCon Italia CfP is open</h2>
   <p>May 28-31 - Bologna</p>
    <p>CfP open until January 6 AoE</p>
    <p>Submit your talk proposal now!</p>
   <p><a href="https://2025.pycon.it/call-for-proposals"> https://pycon.it/call-for-proposals</a>
</section>